src/webviews/api/configuration/appRouter.ts

```diff
@@ -129,6 +275,7 @@ const commonRouter = router({
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const appRouter = router({
     common: commonRouter,
+    demo: demoRouter,
```

Comment:
Should we comment this out with a note or does it need to be enabled?

Reply:
You're right, I'll mark it as a todo in code as well.

It's in for now to simplify demoing it to the team, once we agree on the direction, I'll submit another PR with updates and additional documentation for the new api.